src/app/session/[type]/Steps/helper.ts

```diff
@@ -11,64 +11,42 @@ import {
 } from '@/types';
 import { UseFormReturn } from 'react-hook-form';
 
-export const setGroupPreset = (
-  value: string,
-  apiOptions: ApiFormOptions,
-  updateFormData: (data: Session | ((prevState: Session) => Session)) => void
-) => {
+export const setGroupPreset = (value: string, form: UseFormReturn, apiOptions: ApiFormOptions) => {
   if (!value) return;
 
-  let newDefaultData: Session = {};
+  let newDefaultData: Partial<basicFields> = {};
   switch (value) {
```

Comment:
have a default switch case... if its none of the values.. then 
redirection true
signup false
signin session type
popup false
form names null